influxdb/client_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/segmentio/stats/v4"
 )
 
-func TestClient(t *testing.T) {
```

Comment:
Not sure if we want to keep this. I don't think the code has been exercised in a while.